src/_data/whats-new.yml

```diff
@@ -4,8 +4,148 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Tue Dec 3 09:59:53 2019
+updated: Mon Jan 27 21:17:38 2020
 entries:
+- description: Added `@method` and `@link` information to [Coding Standards](https://devdocs.magento.com/guides/v2.3/coding-standards/docblock-standard-general.html).
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: January 17, 2020
+  link: https://github.com/magento/devdocs/pull/6223
+- description: Updated the [Pro architecture topic](https://devdocs.magento.com/cloud/architecture/pro-architecture.html#pro-cluster-scaling)
+    topic in the _Cloud Guide_ to include information about scaling the architecture.
+  versions: ''
```

Comment:
missing versions

Reply:
it should be 2.x then I guess

Reply:
or it could use the cloud release line versioning, as we do for bundled extensions